templates/skeleton/remix.env.d.ts

```diff
@@ -24,6 +24,8 @@ declare global {
     PRIVATE_STOREFRONT_API_TOKEN: string;
     PUBLIC_STORE_DOMAIN: string;
     PUBLIC_STOREFRONT_ID: string;
+    PUBLIC_CUSTOMER_ACCOUNT_API_CLIENT_ID: string;
+    PUBLIC_CUSTOMER_ACCOUNT_API_URL: string;
```

Comment:
feels a bit funny that we are manually updating these. 
@frandiox does CLI have a type defined when it create the `.env` file already that can be imported?